wlc/main.py

```diff
@@ -271,6 +271,19 @@ def check_result(result, message):
         if not result["result"]:
             raise CommandError(message, result["detail"] if "detail" in result else "")
 
+class ProjectCommand(ObjectCommand):
+    """Wrapper to allow only project objects."""
+
+    def get_object(self):
+        """Return component object."""
+        obj = super(ProjectCommand, self).get_object()
+        if not isinstance(obj, wlc.Project):
+            raise CommandError("Not supported")
+        return obj
+
+    def run(self):
+        """Main execution of the command."""
+        raise NotImplementedError()
```

Comment:
```suggestion

class ProjectCommand(ObjectCommand):
    """Wrapper to allow only project objects."""

    def get_object(self):
        """Return component object."""
        obj = super(ProjectCommand, self).get_object()
        if not isinstance(obj, wlc.Project):
            raise CommandError("Not supported")
        return obj

    def run(self):
        """Main execution of the command."""
        raise NotImplementedError()

```

... or use `black` to format the code.